cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -124,6 +124,18 @@ function FlyoutMenu( element, triggerSel, contentSel, altTriggerSel ) {
     this.dispatchEvent( 'collapseEnd', { target: this } );
   }
 
+  /**
+   * @returns {Object}
+   *   Hash of trigger, alternative trigger, and content DOM references.
+   */
+  function _getDom() {
+    return {
+      altTrigger: _altTriggerDom,
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/61a40782b361155fe2cbbe50a21dce76a7763d6b